### ISSUE TITLE
Try to send macro detector but client crashed

### ISSUE THREAD

**Issue by kaninhot004** [rathena/rathena#7380]
### rAthena Hash

9bf012d29

### Client Date

2022-04-06

### Server Mode

Renewal

### Result

Use /macro_detector -> Char -> Hit another player -> Send -> That player client crashed
4 servers not had red error log.

### Relevant Log Output

```Shell
Module Name: C:\Users\kaifa\Downloads\THE BOX 4\THE BOX 4.exe
Time Stamp: 0x00000000 - Thu Jan  1 07:00:00 1970


Exception Type: 0xc0000005

0x5cee282e	VCRUNTIME140.dll
0x00aba0b5	THE BOX 4.exe
0x007f9b21	THE BOX 4.exe
0x007a9e01	THE BOX 4.exe
0x007792ff	THE BOX 4.exe
0x00779248	THE BOX 4.exe
0x00661c24	THE BOX 4.exe
0x008a8848	THE BOX 4.exe
0x00b473c5	THE BOX 4.exe
0x7638fa29	KERNEL32.DLL
0x772c7bbe	ntdll.dll
0x772c7b8e	ntdll.dll

eax: 0x01121e18	ebx: 0x01121a18
ecx: 0x00000400	edx: 0x00000400
esi: 0x01121a18	edi: 0x00000dc7
ebp: 0x0019b2ec	esp: 0x0019b2c8

stack 0019b2c8 - 0019b6c8
0019B2C8 : 00 04 00 00 00 04 00 00 79 AC AB 00 C7 0D 00 00 
0019B2D8 : 18 1A 12 01 00 04 00 00 48 DB EA 39 D8 D1 FE 57 
0019B2E8 : 00 04 00 00 0C B3 19 00 B5 A0 AB 00 C7 0D 00 00 
0019B2F8 : 00 04 00 00 18 1A 12 01 00 04 00 00 D8 D1 FE 57 
0019B308 : 48 DB EA 39 20 B3 19 00 21 9B 7F 00 18 1A 12 01 
0019B318 : 00 04 00 00 A0 47 D0 74 B0 F5 19 00 01 9E 7A 00 
0019B328 : 14 1A 12 01 18 1A 12 01 00 04 00 00 9E 71 31 C8 
0019B338 : A0 47 D0 74 D8 D1 FE 57 00 00 00 E0 00 00 00 80 
0019B348 : 00 00 00 80 00 00 00 80 00 00 00 80 00 00 00 80 
0019B358 : 00 00 00 80 00 00 00 80 00 00 00 80 00 00 00 80 
0019B368 : 00 00 00 80 00 00 00 80 00 00 00 80 00 00 00 80 
0019B378 : 00 00 00 80 00 00 00 80 00 00 00 80 00 00 00 80 
0019B388 : 00 00 00 80 00 00 00 80 00 00 00 80 00 00 00 80 
0019B398 : 00 00 00 80 00 00 00 80 00 00 00 80 00 00 00 80 
0019B3A8 : 00 00 00 E0 00 00 00 FF 00 00 00 FF 00 00 00 E0 
0019B3B8 : 00 00 00 FF 00 00 00 80 00 00 00 80 00 00 00 80 

Launch Info 
02A8 02A7 02A6 0141 0141 00A4 00A4 0025 
0000 0000 0000 0000 0000 0000 0000 0000 
0000 0000 0000 0000 0000 0000 0000 0000 
0000 0000 0000 0000 0000 0000 0000 0000 

Job : Novice

prontera.rsw 160 184

renderer.cpp 838
```

### Expected Result

Client won't crash when received macro_detector.

### How to Reproduce

Sometime when you send macro_detector to players it will crash client.

![Capture](https://user-images.githubusercontent.com/9493525/197376934-f4f99ada-e263-4637-99c1-2649e7fe09c4.JPG)

### Modifications that may affect results

I used 220x90 bmp 24 bit with black background and simple white text.
Preview images: https://github.com/kaninhot004/rathena/tree/the-box-4/captcha

Is it required to use any specific format?

**Comment by violent01** [comment]
I also encountered this error when I only have 1 captcha image. Make many images as much as you can like 10-15 Images and it will solve your problem as it solved mine. My guess is the client expects different kinds of images and you only provide one.

**Comment by violent01** [comment]
You should definitely use that patch in the exe, I thought you already did that.

**Comment by Lemongrass3110** [comment]
The additional macro detection features by @aleos89 were merged in ee2dcf8.
Please test again and make sure to diff your exe correctly.
